frontend/src/lib/constants.tsx

```diff
@@ -118,6 +118,7 @@ export const FEATURE_FLAGS = {
     EXPERIMENTS_SECONDARY_METRICS: 'experiments-secondary-metrics',
     RECORDINGS_FILTER_EXPERIMENT: 'recording-filters-experiment', // owner: @rcmarron
     DASHBOARD_PERMISSIONS: 'dashboard-permissions', // owner: @Twixes
+    AND_OR_FILTERING: 'and-or-filtering',
```

Comment:
Please add an owner (I guess… you?) so that there's a person responsible for getting rid of the flag when the time comes.

Reply:
ok! i can do that, I left it empty cause technically @EDsCODE owns the project but I guess I'm working on the frontend changes for it 🤔 so not sure :o

Reply:
Add Eric or yourself! :)  (same for secondary-metrics, feel free to add me, or yourself!)